src/api/v1beta1/feature_flags.go

```diff
@@ -276,3 +282,19 @@ func (dk *DynaKube) getFeatureFlagRaw(annotation string) string {
 	}
 	return ""
 }
+
+func (dk *DynaKube) FeatureMaxFailedCsiMountAttempts() int {
+	maxCsiMountAttemptsValue := dk.getFeatureFlagRaw(AnnotationFeatureMaxFailedCsiMountAttempts)
+
+	if maxCsiMountAttemptsValue == "" {
+		return defaultMaxFailedCsiMountAttempts
+	}
+
+	maxCsiMountAttempts, err := strconv.Atoi(maxCsiMountAttemptsValue)
+
+	if err != nil || maxCsiMountAttempts < 0 {
+		return 0
```

Comment:
shouldn't this be also `defaultMaxFailedCsiMountAttempts`?

Reply:
done